Storage/SQL/SQLiteHistory.swift

```diff
@@ -199,19 +237,22 @@ extension SQLiteHistory: BrowserHistory {
     }
 
     public func getSitesByFrecencyWithLimit(limit: Int) -> Deferred<Result<Cursor<Site>>> {
-        return self.getSitesByFrecency(limit: limit, filter: nil)
-    }
+        let groupBy = "GROUP BY \(TableHistory).domain_id "
```

Comment:
This might also need an `ORDER BY sum(frecency)`, to smush together all of the top sites.